app/src/main/res/xml/preferences.xml

```diff
@@ -103,6 +103,15 @@
             android:icon="@drawable/ic_notifications"
             android:summary="@string/rate_limit_summary"/>
     </PreferenceCategory>
+    <PreferenceCategory
+        android:key="widgets"
+        android:title="Widgets">
+        <Preference
+            android:key="manage_entity_state_widgets"
+            android:icon="@drawable/ic_widget"
+            android:title="@string/manage_entity_state_widgets"
+            android:summary="@string/manage_entity_state_widgets_summary" />
```

Comment:
Do we really need one for each?  Maybe just separate the widgets by type with a PreferenceCategory in the next screen rather than having a bunch on the main settings screen?

Reply:
ok will update :) I wasn't sure which route to go